gateway/1.4/modules/ROOT/pages/flex-troubleshoot-debug-logs.adoc

```diff
@@ -0,0 +1,40 @@
+= Troubleshoot with Debug Logs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+When troubleshooting Flex Gateway, you may want to set the runtime loglevel to debug to enable more verbose runtime logs. 
+
+This page only explains how to configure debug runtime logs. To configure debug access logs, set the `level` parameter in your xref:policies-included-message-logging.adoc[Message Logging policy] to `DEBUG`.
+
+== Debug Logs Configuration Example 
+
+For both Connected Mode and Local Mode, you configure debug runtime logs using the following YAML configuration file:
+
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: Configuration
+metadata:
+  name: logging-config
+spec:
+  logging:
+    runtimeLogs:
+      logLevel: debug
+----
+
+
+To configure debug runtime logs, use the previous YAML configuration file when prompted for your configuration details in the following tutorials:
+
+* xref:flex-conn-third-party-logs-config.adoc[Configure External Logs in Connected Mode]
+* xref:flex-local-third-party-logs-config[Configure External Logs in Local Mode]
+
+If you have already configured a logging output, you can use the previous YAML configuration file. To enable debug runtime logs, set `logging.runtimeLogs.logLevel` to `debug`. This does not affect the log output location or access logs.
```

Comment:
Is this necessary? Is the jist that you can change the log level in an existing config file?

What do you mean by "previous YAML configuration file"? The one in the example?